yt/frontends/ramses/field_handlers.py

```diff
@@ -462,7 +462,7 @@ def detect_fields(cls, ds):
 
         if nvar == ndim + 1:
             fields = ["potential"] + [f"{k}-acceleration" for k in "xyz"[:ndim]]
-            ndetected = ndim
+            ndetected = ndim + 1
         else:
             fields = [f"{k}-acceleration" for k in "xyz"[:ndim]]
             ndetected = ndim
```

Comment:
I took a few minutes to double check this and I think there's an obvious way to simplify the if/else altogether
```suggestion
        else:
            fields = [f"{k}-acceleration" for k in "xyz"[:ndim]]
        ndetected = len(fields)
```